php/sync/class-download-sync.php

```diff
@@ -196,7 +196,7 @@ public function download_asset( $attachment_id, $source = null, $date = null ) {
 		require_once ABSPATH . 'wp-admin/includes/media.php';
 		if ( empty( $source ) ) {
 			$cloudinary_id = $this->media->get_cloudinary_id( $attachment_id );
-			$source        = $this->media->cloudinary_url( $attachment_id, array(), array(), $cloudinary_id );
+			$source        = $this->media->cloudinary_url( $attachment_id, array(), array(), $cloudinary_id, false, false );
```

Comment:
@dugajean 

Can you confirm that we still need the last argument, and the previous one?